packages/main/test/specs/MultiComboBox.mobile.spec.js

```diff
@@ -188,6 +188,26 @@ describe("Typeahead", () => {
 
 		assert.strictEqual(await mcb.getProperty("value"), "c", "Value is autocompleted");
 	});
+
+	it("Should open the picker with preselected items only when n-more is clicked", async () => {

```

Comment:
Please rename the test because it is doing something else - it checks that a readonly mcb value can not be changed